modules/osdk-prepare-supported-images.adoc

```diff
@@ -1,56 +1,116 @@
 // Module included in the following assemblies:
 //
 // * operators/operator_sdk/golang/osdk-golang-tutorial.adoc
+// * operators/operator_sdk/ansible/osdk-ansible-tutorial.adoc
+// * operators/operator_sdk/helm/osdk-helm-tutorial.adoc
+
+:osdk_ver: v1.3.0
+ifeval::["{context}" == "osdk-golang-tutorial"]
+:golang:
+:type: Go
+endif::[]
+ifeval::["{context}" == "osdk-ansible-tutorial"]
+:ansible:
+:type: Ansible
+endif::[]
+ifeval::["{context}" == "osdk-helm-tutorial"]
+:helm:
+:type: Helm
+endif::[]
 
 [id="osdk-prepare-supported-images_{context}"]
-= Preparing your Operator use supported images
+= Preparing your Operator to use supported images
 
-Before running your Go-based Operator on {product-title}, update your project to use supported images.
+Before running your {type}-based Operator on {product-title}, update your project to use supported images.
 
 .Procedure
 
+ifdef::golang[]
 . Update the following lines in the project root-level Dockerfile to use supported images.
 
-.. Change the default builder image reference from:
+.. Change the default runner image reference from:
 +
 [source,terminal]
 ----
-golang:1.15
+FROM gcr.io/distroless/static:nonroot
 ----
 +
 to:
 +
 [source,terminal]
 ----
-registry.redhat.io/openshift4/builder:rhel-8-golang-1.15-openshift-4.7
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
 ----
 
-.. Change the default runner image reference from:
+.. Remove the `USER: nonroot:nonroot` directive, as it is not required by the supported image.
+
+. In the `config/default/manager_auth_proxy_patch.yaml` file, change the `image` value from:
 +
 [source,terminal]
 ----
-gcr.io/distroless/static:nonroot
+gcr.io/kubebuilder/kube-rbac-proxy:<tag>
 ----
 +
-to:
+to use the supported image:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-registry.access.redhat.com/ubi8/ubi-minimal:latest
+registry.redhat.io/openshift4/ose-kube-rbac-proxy:v{product-version}
 ----
+endif::[]
```

Comment:
Just to check, is the important admonition that is included at this point for Ansible and Helm not required for the Go procedure?

Reply:
That's my understanding, yes. For the Go procedure, we don't have a builder image to tell them to swap out for, so there's nothing for them to match.